TESTING.md

```diff
@@ -176,6 +176,12 @@ Prebuilt development images can be downloaded from here:
 
 Install the package `qemu-system-x86_64` if you don't have already installed.
 
+
+> [!WARNING]
+> Since the commit https://github.com/libremesh/lime-packages/commit/64b7c01961c6e7bc776f83523aea9264724b3b23 lime packages breaks the qemu image and the interfaces cannot be setted up!
+> This is due the migration to newer version of OpenWRT.
+> To use latest lime packages use a OpenWRT 23 image .
+
```

Comment:
Maybe we can also add a link to a prebuilt OpenWRT 23 image.
```suggestion
> 
> New prebuilt images can be downloaded from here:
> * https://firmware-libremesh.antennine.campiinrete.org/releases/master-ow23/targets/x86/64/libremesh-master-ow23-default-x86-64-generic-rootfs.tar.gz
> * https://firmware-libremesh.antennine.campiinrete.org/releases/master-ow23/targets/x86/64/libremesh-master-ow23-default-x86-64-generic-initramfs-kernel.bin
```

Reply:
Not sure if we should link non official repos here..

Reply:
> Not sure if we should link non official repos here..

also further testing has to be done... Let's leave this text behind

Reply:
Added reference to official repo images